configs/docma.json

```diff
@@ -74,5 +82,8 @@
       }
     }
   },
+  "markdown": {
+    "sanitize": false
+  },
```

Comment:
required to have HTML in markdown files (which we use to display the badges and tram-one logo on the main tram-one.io page, see here: https://tram-one.io/